### PR TITLE
Add location info to PlayerSummary

### PR DESCRIPTION
**Pull request by jankcat**

### CODE REVIEW

File: src/structures/PlayerSummary.js [hunk 1]
```diff
@@ -18,6 +18,9 @@ class PlayerSummary extends Player {
 		this.personaStateFlags = player.personastateflags;
 		this.commentPermission = player.commentpermission;
 		this.visibilityState = player.communityvisibilitystate;
+		this.countryCode = player.loccountrycode;
+		this.stateCode = player.locstatecode;
+		this.cityId = player.loccityid;
```
Comment (xDimGG) [inline src/structures/PlayerSummary.js]: `cityId` -> `cityID`

Comment (jankcat) [reply src/structures/PlayerSummary.js]: Fair play. Done.

**Comment by jankcat** [comment]
The more I think about it, the more `Id` makes the most sense instead of `ID`. 
- Id is not an acronym but an abbreviation. 
- In HTML, it is lowercase as `id`
- JS standard is to Camel Case, so it would make sense to be `someId`, capitalizing the first letter of the new word.
- DotNet style guide dictate `id` or `Id`: https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions
- Unfortunately, JS style guides have not weighed in on this one yet.

**Comment by jankcat** [comment]
Edit: JS has a standard convention of Camel, as seen on commands like getElementById: https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementById

**Comment by jankcat** [comment]
GO is a different language than JS. `Id` makes more sense since it is an abbreviation not an acronym.
The only instances of `ID` on MDN are the deprecated BluetoothDevice API's properties, the old NavigatorID, and vender-specifics like USB.

All the new and experimental APIs are `Id`.
- https://developer.mozilla.org/en-US/docs/Web/API/MediaKeySession/sessionId
- https://developer.mozilla.org/en-US/docs/Web/API/FetchEvent/clientId
- https://developer.mozilla.org/en-US/docs/Web/API/VRDisplay/displayId
- https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/sinkId
- https://developer.mozilla.org/en-US/docs/Web/API/PaymentResponse/requestId
- https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/devtools.inspectedWindow/tabId
- https://developer.mozilla.org/en-US/docs/Web/API/MessageEvent/lastEventId
- https://developer.mozilla.org/en-US/docs/Web/API/TaskAttributionTiming/containerId
- https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/pointerId

If it makes you more comfortable, I can PR all the `ID`s changed to `Id`

**Comment by xDimGG** [comment]
Can you just change it back to cityID so I can merge this pull request without having to take over. It's a personal preference and it's what I prefer. Thanks

**Pull request merged**